proposals/use_base32_encoding_of_long_hash_of_pubkey_plus_checksum_for_address.md

```diff
@@ -0,0 +1,494 @@
+
+If any of these validation steps fails, then the address does not have a valid format.
+
+#### Address Format Validation in Transaction Validation
+
+When a transaction object contains an address, the address format has to be validated during the transaction validation. If the format validation fails, the transaction has to be rejected. The procedure described above can be used for the format validation. Of course, a node may also keep track of all validated addresses and may validate the format of an address by a lookup.
+
+#### Address Format Validation in User Interfaces
+
+This subsection does not contain rules for the protocol, but recommendations for the design of user interfaces to make use of all capabilities of the new address format. The recommendations are:
+
+- When entering an address into an input field, letters should be automatically converted to lowercase letters.
+- Before sending out a transaction that contains an address data field, the address format should be verified. If the validation fails, the transaction should not be sent out.
+
+### Serialization
+
+Whenever an address value has to be serialized, the 20 bytes of the public key hash are taken, i.e. the bytes which are used as the input for the checksum computation. This applies for the serialization of the `recipientID` property of balance transfer transaction and for any other property of a transaction (or block) that contains and address.
+
+### Converting Existing Accounts 
+
+We distinguish between accounts with and without associated public key. Note that an account has an associated key if and only if an outgoing transaction from this account is included in the blockchain.
+
+#### Accounts with Public Key
+
+For accounts with an associated public key, the new address has to be computed and stored such that the account is accessible via the new address.
+
+#### Accounts without Public Key
+
+Funds that were sent to an address of the current format and for which no public key was associated before the implementation of the new address system are not automatically accessible. To make them accessible, a special transaction, called _reclaim transaction_, has to be included in the blockchain. The set of addresses of the current format without associated public key and for which no reclaim transaction was included in the blockchain have to be persisted along with their balances. In the following, we denote this set of addresses by `unregisteredAddresses`.
+
+##### Reclaim Transaction
+
+###### Type
+
+The type of the reclaim transaction is the lowest positive integer that is not used for any other transaction type on the Lisk blockchain (8 at the time of writing).
+
+###### Schema
+
+The `asset` property of a reclaim transaction needs to contain the `amount` property which is of type _string_. The value of `asset.amount` must represent a decimal number and this number must be equal to the balance associated to the address of the current format in Beddows.
```

Comment:
I find it a bit weird to specify that "amount" must be of type _string_. Also, in the serialisation below, we say that it should be serialised as a number. Maybe this is an implementation detail?
@shuse2 could you give your take on this?